spec/models/journal_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+RSpec.describe Journal, type: :model do
+  let!(:journal) do
```

Comment:
Let's learn about Rspec **describe** blocks and **context** blocks and use them where appropriate.